pkg/tsdb/elasticsearch/data_query_test.go

```diff
@@ -1431,7 +1431,6 @@ func TestExecuteElasticsearchDataQuery(t *testing.T) {
 				"query": "foo",
 			}`, from, to)
 			require.NoError(t, err)
-			require.Equal(t, res.Responses["A"].ErrorSource, backend.ErrorSourcePlugin)
```

Comment:
In `experimental/errorsource` if the error did not have error source we have added `backend.ErrorSourcePlugin`. Now, we only add error source when error has source. This is okay because in case there is no error source, we default to plugin error when creating metrics and logs.